flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/TimeFunctionsITCase.java

```diff
@@ -926,7 +926,7 @@ private Stream<TestSetSpec> toTimestampLtzTestCases() {
                                         "yyyy-MM-dd HH:mm:ss",
                                         "Asia/Shanghai"),
                                 "TO_TIMESTAMP_LTZ('2023-01-01 00:00:00', 'yyyy-MM-dd HH:mm:ss', 'Asia/Shanghai')",
-                                LocalDateTime.of(2023, 1, 1, 8, 0, 0)
```

Comment:
reflect behavior change, addressing[ this comment](https://github.com/apache/flink/pull/25763#discussion_r1891197006).